lib/interfaces/ws.js

```diff
@@ -97,9 +97,19 @@ module.exports = function (app) {
             debug('security disallowed update')
             return
           }
+          if (spark.request.position) {
+            // if this has a relativePosition subscription, update the position
+            updatePosition(msg, spark)
+          }
           app.handleMessage(spark.request.source || 'ws', msg)
         }
         if (msg.subscribe) {
+          if (!_.isUndefined(msg.context.relativePosition)) {
+            debug('setting up to store position')
+            spark.request.position = {}
+            msg.context.relativePosition.position = spark.request.position
```

Comment:
`relativePosition` is not very clear, how about `msg.context.subscriber.position`? Ditto for `spark.request.subscriberPosition`.